tools/validator-client-scripts/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "validator-client-scripts"
+version = "0.1.0"
+edition = "2018"
```

Comment:
Humble question, is there a reason why we are not using the 2021 edition?

Reply:
The answer is that this binary was originally written before version 2021 was a thing. But an excellent catch, @tommyv1987 go and update that : ) (all you have to do is change `edition = "2018"` to `edition = "2021"`)

Reply:
resolved in inbound commit